roles/deploy-aws-infra/tasks/main.yml

```diff
@@ -0,0 +1,343 @@
+    subnets:
+      - "{{ new_subnets.subnet.id }}"
+    routes:
+      - dest: "0.0.0.0/0"
+        gateway_id: "{{ new_gw.gateway_id }}"
+
+# Create local facts with new subnet id
+- name: Register new created objects as facts
+  set_fact:
+    aws_subnet: "{{ new_subnets.subnet.id }}"
+
+- name: Create Master and etcd instances
+  ec2:
+    image: "{{ aws_image_name }}"
+    instance_type: m4.xlarge
+    group: "{{ aws_master_sgroups }}"
+    termination_protection: yes
+    key_name: "{{ aws_key_name }}"
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
+    vpc_subnet_id: "{{ aws_subnet }}"
+    exact_count: 1
+    count_tag:
+      Name: "{{ env_id }}-ocp-master-{{ item }}"
+    instance_tags:
+      Name: "{{ env_id }}-ocp-master-{{ item }}"
+      user: sre
+      role: ocp
+      group: masters_aws
+      node_labels: '{"region": "default"}'
+      ocp_role: "{{ master_tag }} {{ etcd_tag}}"
+      env_id: "{{ env_id }}"
+  with_sequence:
+    count="{{ aws_num_masters }}"
+  register: instances_master_etcd
+
+- name: Create Infra instances
+  ec2:
+    image: "{{ aws_image_name }}"
+    instance_type: i3.xlarge
+    group: "{{ aws_infra_sgroups }}"
+    termination_protection: yes
+    key_name: "{{ aws_key_name }}"
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
+    vpc_subnet_id: "{{ aws_subnet }}"
+    exact_count: 1
+    count_tag:
+      Name: "{{ env_id }}-ocp-infra-{{ item }}"
+    instance_tags:
+      Name: "{{ env_id }}-ocp-infra-{{ item }}"
+      user: sre
+      role: ocp
+      group: infra_aws
+      node_labels: '{"region": "infra"}'
+      ocp_role: "{{ infranode_tag }}"
+      env_id: "{{ env_id }}"
+  with_sequence:
+    count="{{ aws_num_infra }}"
+  register: instances_infra
+
+- name: Create Compute instances
+  ec2:
+    image: "{{ aws_image_name }}"
+    instance_type: m4.xlarge
+    group: "{{ aws_node_sgroups }}"
+    termination_protection: yes
+    key_name: "{{ aws_key_name }}"
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
+    vpc_subnet_id: "{{ aws_subnet }}"
+    exact_count: 1
+    count_tag:
+      Name: "{{ env_id }}-ocp-node-{{ item }}"
+    instance_tags:
+      Name: "{{ env_id }}-ocp-node-{{ item }}"
+      user: sre
+      role: ocp
+      group: nodes_aws
+      node_labels: '{"region": "primary"}'
+      ocp_role: "{{ computenode_tag }}"
+      env_id: "{{ env_id }}"
+  with_sequence:
+    count="{{ aws_num_nodes }}"
+  register: instances_compute
+
+- name: Give some time until the instances are running before attaching storage
+  pause:
+    seconds: 30
+
+- name: Add EBS for docker storage
+  ec2_vol:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
+    delete_on_termination: yes
+    device_name: /dev/xvdb
```

Comment:
Let's use a variable and default to `/dev/xvdb`

Reply:
As long as we use the same variable for 'docker_storage_block_device' variable on the inventory file we are fine

Reply:
sounds good to me